### ISSUE TITLE
only running when WP_DEBUG_LOG is true

### ISSUE THREAD

**Issue by leonstafford** [leonstafford/simplerstatic#1]
regardless of plugin's debug mode enabled or not

in WP's debug.log, we're seeing errors from the simple dom class

```

 170 [16-May-2020 03:00:21 UTC] PHP Warning:  preg_match_all(): Compilation failed: invalid range in character class at offset 4 in /htdocs/wp-content/plugins/simplerstatic/vendor/sunra/php-simple-html-dom-parser     /Src/Sunra/PhpSimple/simplehtmldom_1_5/simple_html_dom.php on line 697


...

 272 [16-May-2020 03:01:50 UTC] PHP Warning:  preg_match(): Compilation failed: invalid range in character class at offset 4 in /htdocs/wp-content/plugins/simplerstatic/vendor/sunra/php-simple-html-dom-parser/Src     /Sunra/PhpSimple/simplehtmldom_1_5/simple_html_dom.php on line 1378


...

1554 [16-May-2020 03:11:32 UTC] PHP Warning:  Invalid argument supplied for foreach() in /htdocs/wp-content/plugins/simplerstatic/vendor/sunra/php-simple-html-dom-parser/Src/Sunra/PhpSimple/simplehtmldom_1_5/simp     le_html_dom.php on line 704

**Comment by leonstafford** [comment]
related https://github.com/sunra/php-simple-html-dom-parser/issues/75

**Comment by leonstafford** [comment]
closed by 5523e4bd63631800a9e1f33ce615e1240ae3100b
